internal/controllers/topology/cluster/structuredmerge/dryrun.go

```diff
@@ -54,14 +54,14 @@ func dryRunSSAPatch(ctx context.Context, dryRunCtx *dryRunSSAPatchInput) (bool,
 	// The identifier consists of: gvk, namespace, name and resourceVersion of originalUnstructured
```

Comment:
Sorry slightly unrelated. Just noticed that we create an ssaCache for the Machine controller, but never use it there.
Can you please drop the ssaCache field in the Machine controller (while we're already playing around with the SSA cache)

Reply:
pushed up this change

Reply:
Thx!